lib/validate.js

```diff
@@ -15,6 +15,7 @@ var fs = require('fs'),
   util = require('util'),
   SpecValidator = require('./validators/specValidator'),
   WireFormatGenerator = require('./wireFormatGenerator'),
+  XMsExampleGenerator = require('./xMsExampleGenerator'),
```

Comment:
of course. Let me change it to the `Extractor` to make sure we represent things accurately.